dashboard/lib/widgets/task_icon.dart

```diff
@@ -32,6 +32,13 @@ class TaskIcon extends StatelessWidget {
     // different assets.
     final Color? blendFilter = brightness == Brightness.dark ? Colors.white : null;
 
+    if (qualifiedTask.isFrob) {
+      return Image.asset(
+        'assets/ACX.png',
```

Comment:
Let's change this to a google logo.

These tests aren't just ACX, and no one outside of Google will know what ACX is